### DISCUSSION TITLE
Integrating CSS and GA code directly into the page

### DISCUSSION THREAD

**Discussion by krzysiekpaszek** [thuliteio/doks#154] [Q&A] [answered]
Hi.

Is there an easy way to integrate the final generated CSS code not as a separate file, but inject it into evey pages' head section?
Same with Google Analytics. The ability to have the script inside the html

**Comment by h-enk** [comment, marked as answer]
Inline like so: https://discourse.gohugo.io/t/hugo-inline-css-optimization-for-google-pagespeed/6032/21
